### DISCUSSION TITLE
Resetting an arrow file for merge

### DISCUSSION THREAD

**Discussion by dagarfield** [GreenleafLab/ArchR#1883] [Questions / Documentation]
From time to time, it comes up that we'd like to merge two ArchR projects. A typical use case might be that we want to combine two datasets and re-call peaks or maybe generate a new UMAP. Or sometimes just to get a new featureMatrix from one project rather than multiple. 

To do this in the past I have:

1. Exported a fragment file from each project
2. Created a new ArchR project (and new arrow files) from those fragment files. 

This is fine (works), but slow and a bit awkward if you want to routinely combine multiple projects. 
What'd be nice to be able to do is:

1. Make a new copy of each individual arrowFile (each sample from the different projects)
2. Reset it to ground state -- keep only tile matrix and possible gene-score matrix (ie only cell autonomous data)
3. Build a new ArchR project on these new arrow files

In the current structure, is this feasible?
(assuming same annotation on all files, naturally)

**Comment by dagarfield** [comment]
Actually, can I just do this

```
arrowFiles<-c(getArrowFiles(proj1),getArrowFiles(proj2))
proj <- ArchRProject(
  ArrowFiles = arrowFiles, 
  outputDirectory = "combined_project",
  copyArrows = TRUE 
)
```

And then force over-write peak calls, LSI, etc? (again, assuming identity in the annotations used)

**Reply by rcorces** [reply]
This sounds like a reasonable solution and should in theory work. Another option would be to use the hidden `.dropGroupsFromArrow()` function described in https://github.com/GreenleafLab/ArchR/discussions/843
